src/chunks/storage/mod.rs

```diff
@@ -23,6 +23,9 @@ pub use self::wrappers::{StorageViewWrap, StorageWrap};
 pub trait Storage {
     fn embedding(&self, idx: usize) -> CowArray<f32, Ix1>;
 
+    // Retrieve multiple embeddings.
+    fn embeddings(&self, indices: &[usize]) -> Array2<f32>;
```

Comment:
Did we intentionally not document the `Storage` trait? Should this be a doc-comment or just a inline comment?

Reply:
I think this is an accident :(. Changed to doc comment. We should do a separate PR to document these.

Reply:
Fixed that. Good to go?

Reply:
Yup, good to go